api/github_service/routes/PullRequest.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+const router = express.Router();
+const fetch = require('node-fetch');
+const {
+  CLIENT_ID,
+  CLIENT_SECRET } = require('../../config/config.json').githubAppKeys;
+const {
+  OK,
+  NOT_FOUND
+} = require('../../util/constants').STATUS_CODES;
+
+/**
+ * Call to Github API to retrieve active Pull Requests in a given repository
+ */
+router.get('/', (req, res) => {
```

Comment:
i would merge the two files in the `routes` directory into two `get` requests like

```js
router.get('/getPullRequestsForRepo')

router.get('/getContributorsForRepo')
```